### PR TITLE
Some button css fine tuning

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: style_editor.html [hunk 1]
```diff
@@ -101,32 +101,22 @@
...
-}
-.page-left .button-on:hover {
-  background-color: #ddd;
-  padding: 2px;
-  border: 2px solid dodgerblue;
-}
-.page-left .button-on:active {
-  background-color: dodgerblue;
+  box-shadow: 0 0 2px 1px transparent;
+  cursor: not-allowed;
```
Comment (profezzorn) [inline style_editor.html:105]: Doesn't this screw it up if you're trying to use something other than a mouse for input?
Why did you add it?

Comment (brd55) [reply style_editor.html:105]: I assume he’s talking about keyboard users.  Some browsers can be weird about cursors, but I wouldn’t expect this to throw it off (without ever having tested that cursor)

Comment (NoSloppy) [reply style_editor.html:105]: The only non mouse input selection I can think of is tabbing through buttons.
Interestingly, when disabled, the 'Do Selected Effect' button is skipped over completely, while 'Expand' and 'Layerize' actually get the border highlight but remain greyed out and non-"cllickable" with the space bar. ¯\_(ツ)_/¯
